src/pipeline.ml

```diff
@@ -110,6 +110,64 @@ module Packet_unikernel = struct
 end
 module Build_unikernel = Build.Make(Packet_unikernel)
 
+module Docker_context = struct
```

Comment:
Maybe add a comment about what `Docker_context` does that is different to `Cluster_build`.